graphql/user/mutations.ts

```diff
@@ -27,16 +25,16 @@ class SupportMessage {
     subject: string;
 }
 
-@Resolver((of) => UserType)
+@Resolver(() => UserType)
 export class MutateUserResolver {
-    @Mutation((returns) => Boolean)
+    @Mutation(() => Boolean)
     @Authorized(Role.UNAUTHENTICATED)
     @RateLimit('Email Availability', 50 /* requests per */, 5 * 60 * 60 * 1000 /* 5 hours */)
     async isEmailAvailable(@Arg('email') email: string) {
         return await isEmailAvailable(validateEmail(email));
     }
 
-    @Mutation((returns) => String)
+    @Mutation(() => String)
```

Comment:
using the parameter name to add some meaning to the function is in line with TypeGraphQL's documentation (https://typegraphql.com/docs/resolvers.html) and we followed that pretty consistently. So I would prefer to leave them here.

Reply:
I'll add them back, but tbh I don't see the real benefit of having dangling variables for the sake of explanation. Furthermore, it seems to lead to forgotten arguments/variables as the linter cannot care about them right now.

Reply:
True, also I don't see a reason why it is a function ... Anyways, we can discuss this, but then it should be done consistently across all resolvers